_docs-sources/intro/tool-fundamentals/terraform.md

```diff
@@ -0,0 +1,30 @@
+---
+'title': 'Terraform'
+---
+
+# Terraform
+
+[Terraform](https://www.terraform.io) is an open source _provisioning_ tool that allows you to define and manage as code a
+wide variety of infrastructure (e.g., servers, load balancers, databases, network settings, and so on) across
```

Comment:
```suggestion
[Terraform](https://www.terraform.io) is an open source _provisioning_ tool that allows you to define and manage a
wide variety of infrastructure as code (e.g., servers, load balancers, databases, network settings, and so on) across
```

Reply:
I'm changing this slightly to associate the content in parentheses, but your point is ... on point!

Addressed in 438fd89